src/bin/recursion.rs

```diff
@@ -1,12 +1,13 @@
+use anyhow::Result;
 use std::time::Instant;
 
-use plonky::{recursive_verification_circuit, verify_proof_circuit, BufferGate, Circuit, CircuitBuilder, PartialWitness, Tweedledee, Tweedledum};
+use plonky::{recursive_verification_circuit, verify_proof_circuit, BufferGate, Circuit, CircuitBuilder, Field, PartialWitness, RecursionPublicInputs, Tweedledee, Tweedledum};
 
 const INNER_PROOF_DEGREE_POW: usize = 14;
 const INNER_PROOF_DEGREE: usize = 1 << INNER_PROOF_DEGREE_POW;
 const SECURITY_BITS: usize = 128;
 
-fn main() {
+fn main() -> Result<()> {
```

Comment:
Didn't know about that option, that's nice!

Reply:
Yes it's useful. It also works with tests by the way.